app/assets/javascripts/discourse/app/components/sidebar/section-link-prefix.hbs

```diff
@@ -1,5 +1,5 @@
-{{#if @prefixValue}}
-  <span class={{concat-class "sidebar-section-link-prefix" @prefixType @prefixCSSClass}} style={{@prefixCSS}}>
+{{#if @prefixType}}
+  <span class={{concat-class "sidebar-section-link-prefix" @prefixType @prefixCSSClass}} style={{@prefixColorCSS}}>
```

Comment:
`@prefixColorCSS` isn't really correct considering we're applying it to the `style` attribute we can be anything.